Makefile

```diff
@@ -121,11 +121,14 @@ docker-build-microsoft-vhd:
 docker-wait-pachd:
 	etc/compile/wait.sh pachd_compile
 
-docker-build: docker-build-job-shim docker-build-pachd docker-wait-job-shim docker-wait-pachd
+docker-build: docker-build-job-shim docker-build-pachd docker-wait-job-shim docker-wait-pachd docker-build-netcat
```

Comment:
I think it's probably better to leave this out of the main `docker-build` target since that's people's main development loop and netcat isn't strictly necessary for development. To get it to work on CI you could just call `docker-build-netcat` directly.